app/Core/User/UserSession.php

```diff
@@ -223,7 +223,7 @@ public function isLogged()
     public function getFilters($projectID)
     {
         if (! session_exists('filters:'.$projectID)) {
-            return session_get('user')['filter'] ?: 'status:open';
+            return session_get('user') ? session_get('user')['filter'] ?: 'status:open' : 'status:open';
```

Comment:
Why this condition need to be changed?

Reply:
I the unit tests session_get('user') could be null and the test failed when trying to access ['filter'] of null.

https://wiki.php.net/rfc/notice-for-non-valid-array-container